src/panels/lovelace/editor/config-elements/hui-action-base-card-editor.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  customElement,
+  internalProperty,
+  LitElement,
+  property,
+  query,
+} from "lit-element";
+
+import { fireEvent, HASSDomEvent } from "../../../../common/dom/fire_event";
+import { ActionConfig } from "../../../../data/lovelace";
+import { HomeAssistant } from "../../../../types";
+import { LovelaceCardEditor } from "../../types";
+import { GUIModeChangedEvent } from "../types";
+import type { HuiElementEditor } from "../hui-element-editor";
+import type { EditActionEvent } from "../../components/hui-actions-editor";
+
+import "../hui-element-editor";
+
+@customElement("hui-action-base-card-editor")
+export class HuiActionBaseCardEditor extends LitElement
+  implements LovelaceCardEditor {
+  @property({ attribute: false }) public hass?: HomeAssistant;
+
+  @internalProperty() protected _config?: any;
```

Comment:
Use `T extends BaseCardConfig`
```suggestion
  @internalProperty() protected _config?: T;
```